### PR TITLE
fixed mention style and added view more/less style

### PR DESCRIPTION
**Pull request by iabdousd**

### CODE REVIEW

**Comment by iabdousd** [comment]
@nelsonweze should be easy to review and merge, feel free to share any notes that you have!

**Comment by nelsonweze** [comment]
Hey @iabdousd, Thanks for the PR. I was using `linkStyle` for the viewMoreOrLess texts. Perhaps that wasn't enough for you?

**Comment by iabdousd** [comment]
Oh! It wasn't obvious that you're using it for them. But I still think that it's a good addition, what do you think?

File: lib/src/text_view.dart [hunk 1]
```diff
@@ -90,6 +95,7 @@ class _RichTextViewState extends State<RichTextView> {
               TextSpan(text: ' \u2026'),
               TextSpan(
                   text: _expanded ? widget.viewLessText : widget.viewMoreText,
+                  style: widget.viewMoreLessStyle,
```
Comment (nelsonweze) [inline lib/src/text_view.dart:98]: Hi @iabdousd  sorry for the very late response. 
In that case, it's better to add it on line 106 instead as `style: widget.viewMoreLessStyle ?? linkStyle`

**Pull request closed without merging**